### PR TITLE
[maintenance] - Use symfony runtime component

### PR DESCRIPTION
**Pull request by Ferror**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.12                   |
| Bug fix?        | no                                                       |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no  |
| Related tickets | none                      |
| License         | MIT                                                          |

### CODE REVIEW

**Review by loic425 — APPROVED** [review]

File: src/Kernel.php [hunk 1]
```diff
@@ -33,6 +33,6 @@ protected function getContainerBaseClass(): string
 
     private function isTestEnvironment(): bool
     {
-        return 0 === strpos($this->getEnvironment(), 'test');
+        return str_contains($this->getEnvironment(), 'test');
```
Comment (jakubtobiasz) [inline src/Kernel.php]: We should go with `str_start_with` instead of looking for `test` in the whole env name.

**Review by jakubtobiasz — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -114,6 +114,7 @@
         "symfony/property-info": "^5.4 || ^6.0",
         "symfony/proxy-manager-bridge": "^5.4 || ^6.0",
         "symfony/routing": "^5.4 || ^6.0",
+        "symfony/runtime": "^5.4 || ^6.0",
```
Comment (lchrusciel) [inline composer.json]: Can we move it to the dev requirement? So we won't preinstall it for others?

Comment (jakubtobiasz) [reply composer.json]: @lchrusciel why? It is pointless as the runtime is also for production. Creating and handling in `index.php` whether we are on dev env or not doesn't seem to bring any benefit.

Comment (jakubtobiasz) [reply composer.json]: Primarily, we aim to provide it for Sylius-Standard to allow running Sylius on RoadRunner or FrankenPHP.

Comment (Ferror) [reply composer.json]: @lchrusciel that's true. We should move it to require-dev as we don't depend on it in src. We use it just for testing/development

**Review by TheMilek — APPROVED** [review]

**Review by coldic3 — APPROVED** [review]

**Comment by coldic3** [comment]
Thanks, Zbigniew! :tada:

**Pull request merged**